aiofreepybox/api/home.py

```diff
@@ -0,0 +1,266 @@
+# Home structure : adapter > node > endpoint
+class Home:
+
+    def __init__(self, access):
+        self._access = access
+
+    home_endpoint_value_schema = {
+        'value': object
+    }
+
+    create_home_node_rule_payload_schema = {
+        'iconUrl': '',
+        'id': int,
+        'label': '',
+        'name': '',
+        'role': int,
+        'roleLabel': '',
+        'type': ''
+    }
+
+    node_rule_role_schema = {
+        'node': [int],
+        'role': int
+    }
+
+    node_rule_configuration_data_schema = {
+        'roles': [node_rule_role_schema]
+    }
+
+    sms_number_data_schema = {
+        'description': 'Mon numero',
+        'phoneNumber': '',
+        'smsEnabled': bool(True),
+        'voicemailEnabled': bool(True)
+    }
+
+    sms_validation_data_schema = {
+        'applicationHash': ''
+    }
+
+    sms_number_validation_data_schema = {
+        'validationCode': ''
+    }
+
+    next_pairing_step_payload_schema = {
+        'session': int,
+        'pageid': int,
+        'fields': [object]
+    }
+
+    start_pairing_step_payload_schema = {
+        'nfc': bool(True),
+        'qrcode': bool(),
+        'type': ''
+    }
+
+    stop_pairing_step_payload_schema = {
+        'session': int,
+    }
+
+    async def del_home_adapter(self, home_adapter_id):
+        '''
+        Delete home adapter
+        '''
+        return await self._access.delete(f'home/adapters/{home_adapter_id}')
+
+    async def get_home_adapter(self, home_adapter_id):
+        '''
+        Retrieve a registered home adapter
+        '''
+        return await self._access.get(f'home/adapters/{home_adapter_id}')
+
+    async def get_home_adapters(self):
+        '''
+        Retrieve the list of registered home adapters
+        '''
+        return await self._access.get('home/adapters')
+
+    async def get_camera(self):
+        '''
+        Get camera info
+        '''
+        return await self._access.get('camera')
+
+    async def get_camera_snapshot(self, camera_index=0, size=4, quality=5):
+        '''
+        Get camera snapshot
+        '''
+        fbx_cameras = await self.get_camera()
+        return await self._access.get(fbx_cameras[camera_index]['stream_url'].replace('stream.m3u8', f'snapshot.cgi?size={size}&quality={quality}')[1:])
+
+    async def get_camera_stream_m3u8(self, camera_index=0, channel=2):
```

Comment:
Same here, could you explain what the channel is, and why is it defaulted to 2?

Reply:
Added comment:
        `channel`: 1 is SD, 2 is HD